nomenclature/testing.py

```diff
@@ -56,11 +56,9 @@ def _check_mappings(
     dsd = DataStructureDefinition(path / definitions, dimensions)
     if mappings is None:
```

Comment:
Maybe streamline to 
```python
mappings = mappings or "mappings"
if (path / mappings).is_dir():
         RegionProcessor.from_directory(path / mappings, dsd)
...
```

Reply:
I just implemented your suggestion and to my surprise it broke a unit test `test_cli_missing_mappings_runs`. 
After a little bit of digging if found the that this slightly changes the logic. 
Previously if no mappings directory was given and none exists no error would be raised. Now it would raise an error as after the first line mappings would be "mappings" even if none was provided.
Even though I like your suggestion in terms of readability I'd keep it as is to preserve the logic.

Reply:
Ha, thanks, very helpful!